src/tiles/grid/collection-tile.ts

```diff
@@ -43,27 +45,38 @@ export class CollectionTile extends LitElement {
     const cornerRadiusCss = css`var(--collectionTileCornerRadius, 4px)`;
 
     return css`
+      #collection-image-container {
+        /* width: 18rem; */
+        height: 18.5rem;
+        display: flex;
+        justify-content: center;
```

Comment:
I fiddled with the layout a bit to allow the tile to grow to its container size (currently it's fixed size) and allows the image to stay centered in the cell if the container changes sizes. Just a few `flex` suggestions:

```suggestion
        justify-content: center;
        align-items: center;
        flex: 1;
```

Allows us to resize the cell and your tile will always fit properly:

<img width="731" alt="Screen Shot 2022-02-03 at 10 33 55" src="https://user-images.githubusercontent.com/51138/152407165-2ba42048-16e2-433e-925e-95169b7ec800.png">

Reply:
I omitted align-items:center. The collection image is intentionally aligned near the top (to match up with the alignment of other tiles).